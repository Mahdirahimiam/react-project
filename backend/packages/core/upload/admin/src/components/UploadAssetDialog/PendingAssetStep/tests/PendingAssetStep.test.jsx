import React from 'react';

import { lightTheme, ThemeProvider } from '@strapi/design-system';
import { render as renderTL } from '@testing-library/react';
import { IntlProvider } from 'react-intl';
import { QueryClient, QueryClientProvider } from 'react-query';

import { PendingAssetStep } from '../PendingAssetStep';

jest.mock('../../../../utils/getTrad', () => (x) => x);

jest.mock('@strapi/helper-plugin', () => ({
  ...jest.requireActual('@strapi/helper-plugin'),
}));

const queryClient = new QueryClient({
  defaultOptions: {
    queries: {
      refetchOnWindowFocus: false,
    },
  },
});

describe('PendingAssetStep', () => {
  beforeAll(() => {
    // see https://github.com/testing-library/react-testing-library/issues/470
    Object.defineProperty(HTMLMediaElement.prototype, 'muted', {
      set() {},
    });
  });

  it('snapshots the component with valid cards', () => {
    const assets = [
      {
        source: 'url',
        type: 'image',
        url: 'http://localhost:5000/CPAM.jpg',
        ext: 'jpg',
        mime: 'image/jpeg',
        alt: '',
        name: 'something.jpg',
      },
      {
        source: 'url',
        type: 'doc',
        url: 'http://localhost:5000/MARIAGE%20FRACHET%204.pdf',
        ext: 'pdf',
        mime: 'application/pdf',
        name: 'something.pdf',
      },
      {
        source: 'url',
        type: 'video',
        url: 'http://localhost:5000/mov_bbb.mp4',
        ext: 'mp4',
        mime: 'video/mp4',
        alt: '',
        name: 'something.mp4',
      },
    ];

    const { container } = renderTL(
      <QueryClientProvider client={queryClient}>
        <ThemeProvider theme={lightTheme}>
          <IntlProvider locale="en">
            <PendingAssetStep
              assets={assets}
              onClose={jest.fn()}
              onAddAsset={jest.fn()}
              onEditAsset={jest.fn()}
              onClickAddAsset={jest.fn()}
              onCancelUpload={jest.fn()}
              onRemoveAsset={jest.fn()}
              onUploadSucceed={jest.fn()}
            />
          </IntlProvider>
        </ThemeProvider>
      </QueryClientProvider>
    );

    expect(container).toMatchSnapshot();
  });
});
