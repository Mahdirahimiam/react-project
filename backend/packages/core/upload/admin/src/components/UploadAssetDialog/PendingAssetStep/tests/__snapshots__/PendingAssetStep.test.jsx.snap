// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PendingAssetStep snapshots the component with valid cards 1`] = `
.c9 {
  border: 0;
  -webkit-clip: rect(0 0 0 0);
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  width: 1px;
}

.c4 {
  font-size: 0.875rem;
  line-height: 1.43;
  font-weight: 600;
  color: #32324d;
}

.c14 {
  font-size: 0.75rem;
  line-height: 1.33;
  font-weight: 600;
  color: #32324d;
}

.c15 {
  font-size: 0.75rem;
  line-height: 1.33;
  color: #666687;
}

.c19 {
  font-size: 0.75rem;
  line-height: 1.33;
  font-weight: 600;
  color: #ffffff;
}

.c48 {
  font-weight: 600;
  font-size: 0.6875rem;
  line-height: 1.45;
  text-transform: uppercase;
  color: #666687;
}

.c54 {
  font-size: 0.75rem;
  line-height: 1.33;
  color: #ffffff;
}

.c0 {
  background: #f6f6f9;
  padding-top: 16px;
  padding-right: 20px;
  padding-bottom: 16px;
  padding-left: 20px;
}

.c5 {
  background: #ffffff;
  padding: 8px;
  border-radius: 4px;
  border-color: #dcdce4;
  border: 1px solid #dcdce4;
  cursor: pointer;
}

.c10 {
  padding: 32px;
}

.c16 {
  background: #4945ff;
  padding: 8px;
  padding-right: 16px;
  padding-left: 16px;
  border-radius: 4px;
  border-color: #4945ff;
  border: 1px solid #4945ff;
  cursor: pointer;
}

.c22 {
  background: #ffffff;
  border-radius: 4px;
  border-style: solid;
  border-width: 1px;
  border-color: #eaeaef;
  box-shadow: 0px 1px 4px rgba(33,33,52,0.1);
  height: 100%;
}

.c24 {
  position: relative;
}

.c26 {
  position: start;
}

.c31 {
  position: end;
}

.c37 {
  padding-top: 8px;
  padding-right: 12px;
  padding-bottom: 8px;
  padding-left: 12px;
}

.c40 {
  padding-top: 4px;
}

.c42 {
  padding-top: 4px;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c44 {
  background: #eaeaef;
  padding-right: 8px;
  padding-left: 8px;
  min-width: 20px;
}

.c49 {
  width: 100%;
  height: 5.5rem;
}

.c53 {
  background: #32324d;
  color: #ffffff;
  padding: 4px;
  border-radius: 4px;
  position: absolute;
  right: 4px;
  bottom: 4px;
}

.c2 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c6 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c12 {
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: 32px;
}

.c13 {
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c17 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  gap: 8px;
}

.c27 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c38 {
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c45 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c46 {
  border-radius: 4px;
  height: 1.5rem;
}

.c7 {
  position: relative;
  outline: none;
}

.c7 > svg {
  height: 12px;
  width: 12px;
}

.c7 > svg > g,
.c7 > svg path {
  fill: #ffffff;
}

.c7[aria-disabled='true'] {
  pointer-events: none;
}

.c7:after {
  -webkit-transition-property: all;
  transition-property: all;
  -webkit-transition-duration: 0.2s;
  transition-duration: 0.2s;
  border-radius: 8px;
  content: '';
  position: absolute;
  top: -4px;
  bottom: -4px;
  left: -4px;
  right: -4px;
  border: 2px solid transparent;
}

.c7:focus-visible {
  outline: none;
}

.c7:focus-visible:after {
  border-radius: 8px;
  content: '';
  position: absolute;
  top: -5px;
  bottom: -5px;
  left: -5px;
  right: -5px;
  border: 2px solid #4945ff;
}

.c30 {
  height: 18px;
  min-width: 18px;
  margin: 0;
  border-radius: 4px;
  border: 1px solid #c0c0cf;
  -webkit-appearance: none;
  background-color: #ffffff;
  cursor: pointer;
}

.c30:checked {
  background-color: #4945ff;
  border: 1px solid #4945ff;
}

.c30:checked:after {
  content: '';
  display: block;
  position: relative;
  background: url("data:image/svg+xml,%3csvg%20width='10'%20height='8'%20viewBox='0%200%2010%208'%20fill='none'%20xmlns='http://www.w3.org/2000/svg'%3e%3cpath%20d='M8.55323%200.396973C8.63135%200.316355%208.76051%200.315811%208.83931%200.395768L9.86256%201.43407C9.93893%201.51157%209.93935%201.6359%209.86349%201.7139L4.06401%207.67724C3.9859%207.75755%203.85707%207.75805%203.77834%207.67834L0.13866%203.99333C0.0617798%203.91549%200.0617102%203.79032%200.138504%203.7124L1.16213%202.67372C1.24038%202.59432%201.36843%202.59422%201.4468%202.67348L3.92174%205.17647L8.55323%200.396973Z'%20fill='white'%20/%3e%3c/svg%3e") no-repeat no-repeat center center;
  width: 10px;
  height: 10px;
  left: 50%;
  top: 50%;
  -webkit-transform: translateX(-50%) translateY(-50%);
  -ms-transform: translateX(-50%) translateY(-50%);
  transform: translateX(-50%) translateY(-50%);
}

.c30:checked:disabled:after {
  background: url("data:image/svg+xml,%3csvg%20width='10'%20height='8'%20viewBox='0%200%2010%208'%20fill='none'%20xmlns='http://www.w3.org/2000/svg'%3e%3cpath%20d='M8.55323%200.396973C8.63135%200.316355%208.76051%200.315811%208.83931%200.395768L9.86256%201.43407C9.93893%201.51157%209.93935%201.6359%209.86349%201.7139L4.06401%207.67724C3.9859%207.75755%203.85707%207.75805%203.77834%207.67834L0.13866%203.99333C0.0617798%203.91549%200.0617102%203.79032%200.138504%203.7124L1.16213%202.67372C1.24038%202.59432%201.36843%202.59422%201.4468%202.67348L3.92174%205.17647L8.55323%200.396973Z'%20fill='%238E8EA9'%20/%3e%3c/svg%3e") no-repeat no-repeat center center;
}

.c30:disabled {
  background-color: #dcdce4;
  border: 1px solid #c0c0cf;
}

.c30:indeterminate {
  background-color: #4945ff;
  border: 1px solid #4945ff;
}

.c30:indeterminate:after {
  content: '';
  display: block;
  position: relative;
  color: white;
  height: 2px;
  width: 10px;
  background-color: #ffffff;
  left: 50%;
  top: 50%;
  -webkit-transform: translateX(-50%) translateY(-50%);
  -ms-transform: translateX(-50%) translateY(-50%);
  transform: translateX(-50%) translateY(-50%);
}

.c30:indeterminate:disabled {
  background-color: #dcdce4;
  border: 1px solid #c0c0cf;
}

.c30:indeterminate:disabled:after {
  background-color: #8e8ea9;
}

.c18 {
  height: 2rem;
}

.c18 svg {
  height: 0.75rem;
  width: auto;
}

.c18[aria-disabled='true'] {
  border: 1px solid #dcdce4;
  background: #eaeaef;
}

.c18[aria-disabled='true'] .c3 {
  color: #666687;
}

.c18[aria-disabled='true'] svg > g,
.c18[aria-disabled='true'] svg path {
  fill: #666687;
}

.c18[aria-disabled='true']:active {
  border: 1px solid #dcdce4;
  background: #eaeaef;
}

.c18[aria-disabled='true']:active .c3 {
  color: #666687;
}

.c18[aria-disabled='true']:active svg > g,
.c18[aria-disabled='true']:active svg path {
  fill: #666687;
}

.c18:hover {
  border: 1px solid #7b79ff;
  background: #7b79ff;
}

.c18:active {
  border: 1px solid #4945ff;
  background: #4945ff;
}

.c18 svg > g,
.c18 svg path {
  fill: #ffffff;
}

.c56 {
  height: 2rem;
  border: 1px solid #dcdce4;
  background: #ffffff;
}

.c56 svg {
  height: 0.75rem;
  width: auto;
}

.c56[aria-disabled='true'] {
  border: 1px solid #dcdce4;
  background: #eaeaef;
}

.c56[aria-disabled='true'] .c3 {
  color: #666687;
}

.c56[aria-disabled='true'] svg > g,
.c56[aria-disabled='true'] svg path {
  fill: #666687;
}

.c56[aria-disabled='true']:active {
  border: 1px solid #dcdce4;
  background: #eaeaef;
}

.c56[aria-disabled='true']:active .c3 {
  color: #666687;
}

.c56[aria-disabled='true']:active svg > g,
.c56[aria-disabled='true']:active svg path {
  fill: #666687;
}

.c56:hover {
  background-color: #f6f6f9;
}

.c56:active {
  background-color: #eaeaef;
}

.c56 .c3 {
  color: #32324d;
}

.c56 svg > g,
.c56 svg path {
  fill: #32324d;
}

.c28 > * {
  margin-left: 0;
  margin-right: 0;
}

.c28 > * + * {
  margin-left: 8px;
}

.c29 {
  position: absolute;
  top: 12px;
  left: 12px;
}

.c33 {
  position: absolute;
  top: 12px;
  right: 12px;
}

.c36 {
  margin: 0;
  padding: 0;
  max-height: 100%;
  max-width: 100%;
  object-fit: contain;
}

.c35 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  height: 5.5rem;
  width: 100%;
  background: repeating-conic-gradient(#f6f6f9 0% 25%,transparent 0% 50%) 50% / 20px 20px;
  border-top-left-radius: 4px;
  border-top-right-radius: 4px;
}

.c43 {
  margin-left: auto;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
}

.c47 {
  margin-left: 4px;
}

.c39 {
  word-break: break-all;
}

.c25 {
  border-bottom: 1px solid #eaeaef;
}

.c20 {
  display: grid;
  grid-template-columns: repeat(12,1fr);
  gap: 16px;
}

.c21 {
  grid-column: span 4;
  max-width: 100%;
}

.c8 {
  border-color: #dcdce4;
  height: 2rem;
  width: 2rem;
}

.c8 svg g,
.c8 svg path {
  fill: #8e8ea9;
}

.c8:hover svg g,
.c8:focus svg g,
.c8:hover svg path,
.c8:focus svg path {
  fill: #666687;
}

.c8[aria-disabled='true'] svg path {
  fill: #666687;
}

.c1 {
  border-radius: 4px 4px 0 0;
  border-bottom: 1px solid #eaeaef;
}

.c55 {
  border-radius: 0 0 4px 4px;
  border-top: 1px solid #eaeaef;
}

.c11 {
  overflow: auto;
  max-height: 60vh;
}

.c41 {
  text-transform: uppercase;
}

.c34 {
  opacity: 0;
}

.c34:focus-within {
  opacity: 1;
}

.c23 {
  cursor: pointer;
}

.c23:hover .c32 {
  opacity: 1;
}

.c51 svg {
  font-size: 3rem;
}

.c50 {
  border-radius: 4px 4px 0 0;
  background: linear-gradient(180deg,#ffffff 0%,#f6f6f9 121.48%);
}

.c52 canvas,
.c52 video {
  display: block;
  pointer-events: none;
  max-width: 100%;
  max-height: 5.5rem;
}

@media (max-width:68.75rem) {
  .c21 {
    grid-column: span;
  }
}

@media (max-width:34.375rem) {
  .c21 {
    grid-column: span;
  }
}

<div>
  <form>
    <div
      class="c0 c1"
    >
      <div
        class="c2"
      >
        <h2
          class="c3 c4"
          id="title"
        >
          Add new assets
        </h2>
        <button
          aria-disabled="false"
          class="c5 c6 c7 c8"
          type="button"
        >
          <span
            class="c9"
          >
            Close the modal
          </span>
          <svg
            aria-hidden="true"
            fill="none"
            focusable="false"
            height="1rem"
            viewBox="0 0 24 24"
            width="1rem"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M24 2.417 21.583 0 12 9.583 2.417 0 0 2.417 9.583 12 0 21.583 2.417 24 12 14.417 21.583 24 24 21.583 14.417 12 24 2.417Z"
              fill="#212134"
            />
          </svg>
        </button>
      </div>
    </div>
    <div
      class="c10 c11"
    >
      <div
        class="c12"
      >
        <div
          class="c2"
        >
          <div
            class="c13"
          >
            <span
              class="c3 c14"
            >
              3 assets ready to upload
            </span>
            <span
              class="c3 c15"
            >
              Manage the assets before adding them to the Media Library
            </span>
          </div>
          <button
            aria-disabled="false"
            class="c16 c17 c7 c18"
            type="button"
          >
            <span
              class="c3 c19"
            >
              Add new assets
            </span>
          </button>
        </div>
        <div
          class=""
        >
          <div
            class="c20"
          >
            <div
              class="c21"
            >
              <article
                aria-labelledby=":r0:-title"
                class="c22 c23"
                role="button"
                tabindex="-1"
              >
                <div
                  class="c24 c6 c25"
                >
                  <div>
                    <div
                      class="c26 c27 c28 c29"
                    >
                      <div
                        class=""
                      >
                        <input
                          aria-labelledby=":r0:-title"
                          class="c30"
                          type="checkbox"
                        />
                      </div>
                    </div>
                  </div>
                  <div
                    class="c31 c27 c28 c32 c33 c34"
                  >
                    <span>
                      <button
                        aria-disabled="false"
                        aria-labelledby=":r1:"
                        class="c5 c6 c7 c8"
                        tabindex="0"
                        type="button"
                      >
                        <span
                          class="c9"
                        >
                          Remove from selection
                        </span>
                        <svg
                          aria-hidden="true"
                          fill="none"
                          focusable="false"
                          height="1rem"
                          viewBox="0 0 24 24"
                          width="1rem"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M3.236 6.149a.2.2 0 0 0-.197.233L6 24h12l2.96-17.618a.2.2 0 0 0-.196-.233H3.236ZM21.8 1.983c.11 0 .2.09.2.2v1.584a.2.2 0 0 1-.2.2H2.2a.2.2 0 0 1-.2-.2V2.183c0-.11.09-.2.2-.2h5.511c.9 0 1.631-1.09 1.631-1.983h5.316c0 .894.73 1.983 1.631 1.983H21.8Z"
                            fill="#32324D"
                          />
                        </svg>
                      </button>
                    </span>
                    <span>
                      <button
                        aria-disabled="false"
                        aria-labelledby=":r3:"
                        class="c5 c6 c7 c8"
                        tabindex="0"
                        type="button"
                      >
                        <span
                          class="c9"
                        >
                          Edit
                        </span>
                        <svg
                          aria-hidden="true"
                          fill="none"
                          focusable="false"
                          height="1rem"
                          viewBox="0 0 24 24"
                          width="1rem"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            clip-rule="evenodd"
                            d="M23.604 3.514c.528.528.528 1.36 0 1.887l-2.622 2.607-4.99-4.99L18.6.396a1.322 1.322 0 0 1 1.887 0l3.118 3.118ZM0 24v-4.99l14.2-14.2 4.99 4.99L4.99 24H0Z"
                            fill="#212134"
                            fill-rule="evenodd"
                          />
                        </svg>
                      </button>
                    </span>
                  </div>
                  <div
                    class="c35"
                  >
                    <img
                      alt="something.jpg"
                      aria-hidden="true"
                      class="c36"
                      src="http://localhost:5000/CPAM.jpg"
                    />
                  </div>
                </div>
                <div
                  class="c37"
                >
                  <div
                    class="c38"
                  >
                    <div
                      class="c39"
                    >
                      <div
                        class="c40"
                      >
                        <h2
                          class="c3 c14"
                          id=":r0:-title"
                        >
                          something.jpg
                        </h2>
                      </div>
                      <div
                        class="c3 c15"
                      >
                        <span
                          class="c41"
                        >
                          jpg
                        </span>
                      </div>
                    </div>
                    <div
                      class="c42 c27"
                    >
                      <div
                        class="c43"
                      >
                        <div
                          class="c44 c45 c46 c47"
                        >
                          <span
                            class="c3 c48"
                          >
                            Image
                          </span>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </article>
            </div>
            <div
              class="c21"
            >
              <article
                aria-labelledby=":r5:-title"
                class="c22 c23"
                role="button"
                tabindex="-1"
              >
                <div
                  class="c24 c6 c25"
                >
                  <div>
                    <div
                      class="c26 c27 c28 c29"
                    >
                      <div
                        class=""
                      >
                        <input
                          aria-labelledby=":r5:-title"
                          class="c30"
                          type="checkbox"
                        />
                      </div>
                    </div>
                  </div>
                  <div
                    class="c31 c27 c28 c32 c33 c34"
                  >
                    <span>
                      <button
                        aria-disabled="false"
                        aria-labelledby=":r6:"
                        class="c5 c6 c7 c8"
                        tabindex="0"
                        type="button"
                      >
                        <span
                          class="c9"
                        >
                          Remove from selection
                        </span>
                        <svg
                          aria-hidden="true"
                          fill="none"
                          focusable="false"
                          height="1rem"
                          viewBox="0 0 24 24"
                          width="1rem"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M3.236 6.149a.2.2 0 0 0-.197.233L6 24h12l2.96-17.618a.2.2 0 0 0-.196-.233H3.236ZM21.8 1.983c.11 0 .2.09.2.2v1.584a.2.2 0 0 1-.2.2H2.2a.2.2 0 0 1-.2-.2V2.183c0-.11.09-.2.2-.2h5.511c.9 0 1.631-1.09 1.631-1.983h5.316c0 .894.73 1.983 1.631 1.983H21.8Z"
                            fill="#32324D"
                          />
                        </svg>
                      </button>
                    </span>
                    <span>
                      <button
                        aria-disabled="false"
                        aria-labelledby=":r8:"
                        class="c5 c6 c7 c8"
                        tabindex="0"
                        type="button"
                      >
                        <span
                          class="c9"
                        >
                          Edit
                        </span>
                        <svg
                          aria-hidden="true"
                          fill="none"
                          focusable="false"
                          height="1rem"
                          viewBox="0 0 24 24"
                          width="1rem"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            clip-rule="evenodd"
                            d="M23.604 3.514c.528.528.528 1.36 0 1.887l-2.622 2.607-4.99-4.99L18.6.396a1.322 1.322 0 0 1 1.887 0l3.118 3.118ZM0 24v-4.99l14.2-14.2 4.99 4.99L4.99 24H0Z"
                            fill="#212134"
                            fill-rule="evenodd"
                          />
                        </svg>
                      </button>
                    </span>
                  </div>
                  <div
                    class="c49 c6 c50"
                  >
                    <span
                      class="c51"
                    >
                      <svg
                        aria-label="something.pdf"
                        fill="none"
                        height="1rem"
                        viewBox="0 0 24 33"
                        width="1rem"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          d="M23.305 8.126 16.39.749c-.443-.472-1.043-.741-1.643-.741H2.337c-.632 0-1.2.27-1.642.74A2.529 2.529 0 0 0 0 2.5v27.015c0 1.381 1.042 2.493 2.337 2.493h19.326c1.295 0 2.337-1.112 2.337-2.493V9.877c0-.64-.253-1.28-.695-1.751Zm-1.326.606h-5.116c-.568 0-1.042-.505-1.042-1.112V2.197l6.158 6.535Zm-.316 21.827H2.337c-.537 0-.98-.471-.98-1.044V2.5c0-.269.096-.538.285-.74a.949.949 0 0 1 .695-.304h12.126V7.62c0 1.415 1.074 2.56 2.4 2.56h5.78v19.335c0 .573-.443 1.044-.98 1.044Z"
                          fill="#D9822F"
                        />
                        <path
                          clip-rule="evenodd"
                          d="M13.964 20.095c.344.252.781.522 1.312.811a16.714 16.714 0 0 1 1.827-.1c1.53 0 2.451.236 2.764.709.166.212.177.463.031.753 0 .01-.005.02-.016.029l-.03.029v.014c-.063.367-.433.55-1.11.55-.5 0-1.098-.096-1.795-.29a11.908 11.908 0 0 1-2.03-.767c-2.3.232-4.341.633-6.121 1.202-1.593 2.53-2.852 3.795-3.779 3.795a.965.965 0 0 1-.437-.102l-.375-.174a1.668 1.668 0 0 0-.093-.072c-.105-.096-.136-.27-.094-.521.094-.386.385-.828.874-1.325.49-.498 1.177-.963 2.061-1.398.146-.087.266-.058.36.087.02.02.03.039.03.058a34.703 34.703 0 0 0 1.672-2.853c.707-1.313 1.249-2.578 1.623-3.794a10.952 10.952 0 0 1-.476-2.31c-.067-.748-.034-1.364.102-1.847.114-.386.333-.579.655-.579h.344c.24 0 .422.072.547.217.187.203.234.531.14.985a.307.307 0 0 1-.062.116.35.35 0 0 1 .015.116v.434c-.02 1.188-.093 2.115-.218 2.78.572 1.584 1.332 2.733 2.28 3.447ZM4.97 26.047c.542-.232 1.255-.995 2.14-2.289-.531.387-.987.792-1.367 1.217-.38.425-.637.782-.773 1.072Zm6.183-11.412c-.135-.868-.125-1.506.032-1.911v-.03c.01-.009.015-.019.015-.028a.798.798 0 0 1 .203.521c0 .01.005.024.016.044 0 .01.005.019.015.029a.317.317 0 0 0-.062.116c-.073.386-.11.593-.11.622l-.109.637Zm-1.904 7.66a24.288 24.288 0 0 1 4.434-1.173 2.398 2.398 0 0 1-.203-.137 2.761 2.761 0 0 1-.25-.196c-.79-.647-1.452-1.496-1.983-2.549-.28.83-.713 1.782-1.296 2.854-.312.54-.546.94-.702 1.201Zm7.9-.58c1.208 0 1.937.117 2.187.349.02.019.031.033.031.043-.042.01-.135.014-.281.014-.5 0-1.145-.135-1.936-.405Z"
                          fill="#D9822F"
                          fill-rule="evenodd"
                        />
                      </svg>
                    </span>
                  </div>
                </div>
                <div
                  class="c37"
                >
                  <div
                    class="c38"
                  >
                    <div
                      class="c39"
                    >
                      <div
                        class="c40"
                      >
                        <h2
                          class="c3 c14"
                          id=":r5:-title"
                        >
                          something.pdf
                        </h2>
                      </div>
                      <div
                        class="c3 c15"
                      >
                        <span
                          class="c41"
                        >
                          pdf
                        </span>
                      </div>
                    </div>
                    <div
                      class="c42 c27"
                    >
                      <div
                        class="c43"
                      >
                        <div
                          class="c44 c45 c46 c47"
                        >
                          <span
                            class="c3 c48"
                          >
                            Doc
                          </span>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </article>
            </div>
            <div
              class="c21"
            >
              <article
                aria-labelledby=":ra:-title"
                class="c22 c23"
                role="button"
                tabindex="-1"
              >
                <div
                  class="c24 c6 c25"
                >
                  <div>
                    <div
                      class="c26 c27 c28 c29"
                    >
                      <div
                        class=""
                      >
                        <input
                          aria-labelledby=":ra:-title"
                          class="c30"
                          type="checkbox"
                        />
                      </div>
                    </div>
                  </div>
                  <div
                    class="c31 c27 c28 c32 c33 c34"
                  >
                    <span>
                      <button
                        aria-disabled="false"
                        aria-labelledby=":rb:"
                        class="c5 c6 c7 c8"
                        tabindex="0"
                        type="button"
                      >
                        <span
                          class="c9"
                        >
                          Remove from selection
                        </span>
                        <svg
                          aria-hidden="true"
                          fill="none"
                          focusable="false"
                          height="1rem"
                          viewBox="0 0 24 24"
                          width="1rem"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M3.236 6.149a.2.2 0 0 0-.197.233L6 24h12l2.96-17.618a.2.2 0 0 0-.196-.233H3.236ZM21.8 1.983c.11 0 .2.09.2.2v1.584a.2.2 0 0 1-.2.2H2.2a.2.2 0 0 1-.2-.2V2.183c0-.11.09-.2.2-.2h5.511c.9 0 1.631-1.09 1.631-1.983h5.316c0 .894.73 1.983 1.631 1.983H21.8Z"
                            fill="#32324D"
                          />
                        </svg>
                      </button>
                    </span>
                    <span>
                      <button
                        aria-disabled="false"
                        aria-labelledby=":rd:"
                        class="c5 c6 c7 c8"
                        tabindex="0"
                        type="button"
                      >
                        <span
                          class="c9"
                        >
                          Edit
                        </span>
                        <svg
                          aria-hidden="true"
                          fill="none"
                          focusable="false"
                          height="1rem"
                          viewBox="0 0 24 24"
                          width="1rem"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            clip-rule="evenodd"
                            d="M23.604 3.514c.528.528.528 1.36 0 1.887l-2.622 2.607-4.99-4.99L18.6.396a1.322 1.322 0 0 1 1.887 0l3.118 3.118ZM0 24v-4.99l14.2-14.2 4.99 4.99L4.99 24H0Z"
                            fill="#212134"
                            fill-rule="evenodd"
                          />
                        </svg>
                      </button>
                    </span>
                  </div>
                  <div
                    class="c35"
                  >
                    <div
                      class="c27"
                    >
                      <div
                        class="c52"
                      >
                        <figure
                          class=""
                        >
                          <video
                            crossorigin="anonymous"
                            src="http://localhost:5000/mov_bbb.mp4"
                          >
                            <source
                              type="video/mp4"
                            />
                          </video>
                          <figcaption
                            class="c9"
                          >
                            something.mp4
                          </figcaption>
                        </figure>
                      </div>
                    </div>
                  </div>
                  <time
                    class="c53"
                  >
                    <span
                      class="c3 c54"
                    >
                      ...
                    </span>
                  </time>
                </div>
                <div
                  class="c37"
                >
                  <div
                    class="c38"
                  >
                    <div
                      class="c39"
                    >
                      <div
                        class="c40"
                      >
                        <h2
                          class="c3 c14"
                          id=":ra:-title"
                        >
                          something.mp4
                        </h2>
                      </div>
                      <div
                        class="c3 c15"
                      >
                        <span
                          class="c41"
                        >
                          mp4
                        </span>
                      </div>
                    </div>
                    <div
                      class="c42 c27"
                    >
                      <div
                        class="c43"
                      >
                        <div
                          class="c44 c45 c46 c47"
                        >
                          <span
                            class="c3 c48"
                          >
                            Video
                          </span>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </article>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c0 c55"
    >
      <div
        class="c2"
      >
        <div
          class="c17"
        >
          <button
            aria-disabled="false"
            class="c16 c17 c7 c56"
            type="button"
          >
            <span
              class="c3 c19"
            >
              cancel
            </span>
          </button>
        </div>
        <div
          class="c17"
        >
          <button
            aria-disabled="false"
            class="c16 c17 c7 c18"
            type="submit"
          >
            <span
              class="c3 c19"
            >
              Upload 3 assets to the library
            </span>
          </button>
        </div>
      </div>
    </div>
  </form>
  <div
    class="c9"
  >
    <p
      aria-live="polite"
      aria-relevant="all"
      id="live-region-log"
      role="log"
    />
    <p
      aria-live="polite"
      aria-relevant="all"
      id="live-region-status"
      role="status"
    />
    <p
      aria-live="assertive"
      aria-relevant="all"
      id="live-region-alert"
      role="alert"
    />
  </div>
</div>
`;
